src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/compute.cpp

```diff
@@ -327,6 +374,40 @@ RMSNorm::RMSNorm(const std::shared_ptr<ov::npuw::online::Snapshot>& snapshot, co
     register_matcher(std::make_shared<opp::Matcher>(multiply2, "TagRMSNorm"), std::move(callback));
 }
 
+// TODO: visualize
+RMSNorm2::RMSNorm2(const std::shared_ptr<ov::npuw::online::Snapshot>& snapshot, const std::string& isol_tag) {
+    auto hadd = opp::wrap_type<ov::op::v1::Add>({opp::any_input(), opp::any_input()});
+    auto power = opp::wrap_type<ov::op::v1::Power>({hadd, opp::any_input()});
+    auto reduce = opp::wrap_type<ov::op::v1::ReduceSum>({power, opp::any_input()});
+    auto sqrt = opp::wrap_type<ov::op::v0::Sqrt>({reduce});
+    auto div = opp::wrap_type<ov::op::v1::Divide>({hadd, sqrt});
+    auto multiply = opp::wrap_type<ov::op::v1::Multiply>({opp::any_input(), div});
```

Comment:
How different it is compared to the normal RMSNorm? Can those be combined via optional or choice over add vs reduce, etc?

Reply:
Not possible, the first add connects to different nodes